composer/cli/launcher.py

```diff
@@ -260,7 +261,7 @@ def _launch_processes(
     command_mode: bool,
     training_script: str,
     stdout_file_format: str,
-    stderr_file_format: str,
+    stderr_file_format: Union[str, None],
```

Comment:
Why is this needed? Is it to force it to stdout?

Reply:
Yea, if no stderr file format is provided then we default to piping stderr to stdout